automated install/basic-install.sh

```diff
@@ -1224,7 +1224,8 @@ version_check_dnsmasq() {
     # Local, named variables
     local dnsmasq_conf="/etc/dnsmasq.conf"
     local dnsmasq_conf_orig="/etc/dnsmasq.conf.orig"
-    local dnsmasq_pihole_id_string="# Dnsmasq config for Pi-hole's FTLDNS"
+    local dnsmasq_pihole_id_string="addn-hosts=/etc/pihole/gravity.list"
+    local dnsmasq_pihole_id_string2="# Dnsmasq config for Pi-hole's FTLDNS"
```

Comment:
This second string is not needed. There should never be a `/etc/dnsmasq.conf` with this string.

Reply:
You're right, I misunderstood the purpose of this routine